src/utils/globalConfig/getGlobalConfig.js

```diff
@@ -1,7 +1,10 @@
 const path = require('path')
+const { tryRequireSync } = require('@keg-hub/jsutils/src/node')
```

Comment:
Eventually this will be removed and replaced with the code from `cli-utils` repo => https://github.com/simpleviewinc/keg-cli/blob/master/repos/cli-utils/src/task/getKegGlobalConfig.js
  * Not sure if it's easy enough to do that now, rather then commit these changes
  * If it's a pain don't worry about it, just figured I mention it

Reply:
It looks to me like `getGlobalConfig` does some things that `getKegGlobalConfig` does not, like updating a global config cache. Is it intended for `getKegGlobalConfig` to do less?

Reply:
So `getKegGlobalConfig` and `getGlobalConfig` do the exact same thing. 

The reason it's cached in `getGlobalConfig` is for
* performance improvements, so it doesn't have to re-read from disk
* Allow it to be called from anywhere and still return the same config object.

For now don't worry about it. The best option is to add the caching to `getKegGlobalConfig`, then remove `getGlobalConfig`. But that can wait